copyright

```diff
@@ -8,7 +8,8 @@ Copyright: Michael Zahniser <mzahniser@gmail.com>
            endless-sky contributors (see credits.txt and changelog)
 License: GPL-3+
 
-Files: images/*
+Files:
+ images/*
```

Comment:
Why introduce this churn, changing these perfectly valid copyright specifications? Whitespace-separated lists are allowed to be on one line.

```suggestion
Files: images/*
```

Reply:
OK, will do. Is this specified somewhere in our documentation?

Reply:
Not ours: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/

Reply:
Alright. Thanks for providing a link.